services/data/earlyVoting/GA/knownLocations/WILKES.json

```diff
@@ -0,0 +1,11 @@
+{
+	"23 COURT STREET ROOM 113 WASHINGTON GA 30673": {
+		"location_name": "BOARD OF REGISTRARS",
+		"original_address": "23 COURT STREET ROOM 113 WASHINGTON, GA 30673",
+		"location_type": "APPROXIMATE/ROOFTOP",
+		"formatted_address": "23 E Court St #222, Washington, GA 30673, USA",
+		"lat": 33.738177,
+		"lng": -82.739013,
+		"corrected": true
```

Comment:
For the most part the geocoding is automatic but in some cases, like this address, Google cannot find it. For this one and others, I edited the address to match the early voting site. I used google maps and sometimes the local county offices to make the corrections. In all cases I added the "corrected" flag to the data.
There are a total of 22 hand corrected locations out of a total of 258 know locations.
So that's an 8.5% rate of hand corrections. In this case removing "ROOM 113" form the address would have given a better result. Future code changes should try to reduce that 8.5% rate as much as possible.